gen/dcos-config.yaml

```diff
@@ -379,6 +379,31 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
+  - path: /etc/dcos-metrics-config.yaml
+    content: |
+      #######################################
+      # Configuration file for dcos-metrics #
+      #######################################
+
+      # Collector configuration
+      collector:
+        http_profiler: false
+
+        node:
+          poll_period: 15
+
+        # DC/OS (Mesos) agent configuration
+        mesos_agent:
```

Comment:
This same config file is used both for masters and agents. Does metrics magically select whether to ignore this section if it's an agent? Or should we be outputting two config files, one for agents which says to ask the agent, one for masters which says not to ask it currently.